docs/platforms/javascript/common/sourcemaps/uploading/angular-nx.mdx

```diff
@@ -93,7 +105,25 @@ module.exports = {
 
 Learn more about configuring the plugin in our [Sentry webpack plugin documentation](https://www.npmjs.com/package/@sentry/webpack-plugin).
 
-### 3. Upload
+### esbuild
+
+Install the Sentry esbuild plugin:
+
+```shell {tabTitle:npm}
+npm install --save-dev @sentry/esbuild-plugin
+```
+
+```shell {tabTitle:Yarn}
+yarn add -D  @sentry/esbuild-plugin
+```
+
+```shell {tabTitle:pnpm}
+pnpm add -D @sentry/esbuild-plugin
+```
+
+Then, follow the [official Nx documentation](https://nx.dev/nx-api/angular/executors/browser-esbuild#examples) to register the Sentry esbuild plugin (`@sentry/esbuild-plugin`) in your `project.json` file.
```

Comment:
If it's not too much work, could you add a code sample to register the plugin like you have for webpack? This would make it so much easier for users!

Reply:
yup good callout, thx! added a small example. 

side-note: This way of registering a plugin really is a shitty API :(